src/storage.test.ts

```diff
@@ -4,10 +4,14 @@ function functionThatThrowsAnError() {
 	throw new Error('bang');
 }
 
+type StorageName = 'local' | 'session';
+const LOCAL: StorageName = 'local';
+const SESSION: StorageName = 'session';
+
 describe.each([
-	['local', storage.local, global.localStorage],
-	['session', storage.session, global.sessionStorage],
-])('storage.%s', (title, implementation, native) => {
+	[LOCAL, storage[LOCAL], global.localStorage],
+	[SESSION, storage[SESSION], global.sessionStorage],
+])('storage.%s', (name, implementation, native) => {
```

Comment:
```suggestion
	[storage[LOCAL], global.localStorage],
	[storage[SESSION], global.sessionStorage],
])('%s', (name, implementation, native) => {
```

Reply:
`name` is needed further down for some dynamic imports too

Reply:
Couldn’t spot it. Ignore me!